tests/c_languages/CPPLintBearTest.py

```diff
@@ -7,6 +7,11 @@
 }
 """
 
+test_file2 = ('\nint main() {\n' +
+              '    int x;\n' +
+              '    x = 3;' * 100 + '\n'
+              '    return 0;\n}\n')
```

Comment:
optional: for the sake of readability you can move the `}`-part onto a next line ;)